issue: Delete Org Session Failure

JediKev opened the pull request:
This addresses an issue on the Forum where deleting an Organization causes the session to fail. This is due to the Organization QuerySet being stored in the session and not being cleared out when deleted from the system. This causes the system to try to refetch the object from the database which it's not there as it's been deleted and therefore causes the session to crash.

JediKev committed:
issue: Delete Org Session Failure

This addresses an issue on the Forum where deleting an Organization causes
the session to fail. This is due to the Organization QuerySet being stored
in the session and not being cleared out when deleted from the system. This
causes the system to try to refetch the object from the database which it's
not there as it's been deleted and therefore causes the session to crash.

JediKev commented:
Jk 😅 This was raised here:
#4628

protich commented on include/class.organization.php:
```diff
@@ -471,6 +471,9 @@ function delete() {
             return false;
 
+        // Clear organization from session to avoid refetch failure
+        unset($_SESSION[':Q:orgs'], $_SESSION[':O:tickets']);
```
Why are you clearing tickets here?

JediKev replied:
The org is stored there as well. Maybe I shoul just clear the org from tickets? I can do something like:
```
unset($_SESSION[':Q:orgs'], $_SESSION[':O:tickets'][':O:'.$id]);
```

protich replied:
Let me try replicating the issue and revert back to you with thoughts...

JediKev replied:
I can also send you a dump of my session if you want.

JediKev committed:
orm: Refetch Failure

This addresses an ORM failure where upon refetching an object from the
database that no longer exists the system crashes hard. We use the `one()`
method which expects one result, if there is no result it throws a
`DoesNotExist` exception. We are not catching the exception properly which
crashes osTicket and forces you to clear the session via browser or
database.
